test/support/conn_case.ex

```diff
@@ -22,10 +22,12 @@ defmodule ElixirBoilerplateWeb.ConnCase do
 
   using do
     quote do
-      # Import conveniences for testing with connections
-      use Phoenix.ConnTest
       import ElixirBoilerplateWeb.Router.Helpers
```

Comment:
I think `ElixirBoilerplateWeb.Router.Helpers` should fall under the "conveniences for testing with connections" group.

Reply:
I was unsure due to the alphabetic order thing since they all are imports now instead of `use` and `import`.

I will try it and see if our checks are good with it!